bin/deploy.sh

```diff
@@ -274,6 +285,7 @@ if [ ! -z "$DRYRUN" ]; then
 fi
 
 create_namespace
+create_route53_records
```

Comment:
The intention of deploy.sh is to be cloud-provider agnostic - so I don't think we want to put this call in here

Reply:
Working on moving this to eks-create-ingress-controller.sh and will update the pull request.